### ISSUE TITLE
default job limits aren't documented

### ISSUE THREAD

**Issue by drj11** [rcgsheffield/iceberg_software#363]
I was looking at http://docs.iceberg.shef.ac.uk/en/latest/using-iceberg/sge.html#sge-interactive hoping to find out what the default job limits for CPU time and RAM occupancy were, but I couldn't find them.

Should document what the defaults are (for batch and interactive) and what maximum is allowed by the scheduler.

**Comment by willfurnass** [comment]
Definitely.  There is [some info in the old documentation](https://www.shef.ac.uk/wrgrid/using/requirements) but this needs consolidating here.

**Comment by willfurnass** [comment]
We still really need this.  Wanted to point a user towards the default run time limit for batch jobs (which is actually `h_rt=08:00:00`) but couldn't find it on http://docs.hpc.shef.ac.uk/en/latest/hpc/scheduler/sge.html
